docs/extensions-plugins/sublime.mdx

```diff
@@ -0,0 +1,160 @@
+---
+title: Pieces for Developers Sublime Plugin
+description: Learn about the Pieces Sublime Extension, how it functions, and all you can do with the Sublime Plugin.
+---
+
+import { pieces_app } from "/src/lib/const";
+import CTAButton from "/src/components/CTAButton";
+import GridFlexRow from "/src/components/GridFlexRow";
+import Video from "../../src/components/Video";
+
+# [Pieces for Developers](https://pieces.app) | Sublime Plugin
+
+![Pieces Sublime Plugin](https://github.com/Bishoy-at-pieces/plugin_sublime/assets/167128903/272cf4cc-60ff-4bd4-8701-3e3feb38a8b1)
+
+Please make sure you have [**Pieces OS**](https://docs.pieces.app/installation-getting-started/what-am-i-installing) installed to run the Package.
+
+## Getting Started with the Pieces Sublime Plugin
+
+Welcome to the **Pieces Sublime Plugin**!
+
+The Pieces Sublime plugin is your gateway to managing your saved materials in Pieces directly from within Sublime Text. Seamlessly interact with Pieces OS and enhance your workflow like never before.
+
+## Features
+### Configuration
+
+Before diving in, let's ensure everything is set up to your liking. You can customize the LLM model or the host by navigating to:
+`Preferences > Package Settings > Pieces > Settings`.
+You can also open the settings from the command palette `Pieces: Open Settings`.
+
+### Right Click to Save to Pieces
+
+Saving a new snippet in Pieces is a breeze! Just select the code you want to save, right-click to open the context menu, and choose `Pieces > Save to Pieces`. Your code is now safely stored!
+
+<Video type={'gif'} src={'https://storage.googleapis.com/pieces_multimedia/PROMOTIONAL/PIECES_FOR_DEVELOPERS/SUBLIME/create.gif'} alt={'Save to Pieces in Sublime'} />
+
+### Engage with the Pieces Copilot
+
+Ever wished for a personal coding assistant? Look no further! With Pieces Copilot, you can have an AI-powered assistant right at your fingertips. Simply open the command palette and select `Pieces: Copilot`. From there, you can ask anything whether it's for coding help, debugging, or even generating snippets to save to Pieces with some clicks.
+
+<Video type={'gif'} src={'https://storage.googleapis.com/pieces_multimedia/PROMOTIONAL/PIECES_FOR_DEVELOPERS/SUBLIME/copilot.gif'} alt={'Copilot in Sublime'} />
+
+### Copilot Ask
+
+Need some help from the Pieces Copilot? Whether it's commenting, fixing a bug, or adding a docstring, just select the code, right-click to open the context menu, and choose `Pieces > Ask Copilot`. Let Copilot do the heavy lifting for you!
```

Comment:
Given the fact that we have multiple Ask methods - can you mention that you can ask about both current file and project